### PR TITLE
Add alternative compression with uglify-js 3

### PR DESCRIPTION
**Pull request by rwaldron**
Similar to the way we used to do it with different compressors, but now using uglify-es and uglify-js

### CODE REVIEW

**Comment by tessel-highfive** [comment]
r? @hipsterbrown

(tessel-highfive has picked a reviewer for you, use r? to override)

File: lib/tessel/deployment/lists/javascript.js [hunk 1]
```diff
@@ -54,7 +54,14 @@ module.exports = {
     },
   },
   compressionBlacklist: {
-    bindings: true,
-    base64id: true,
```
Comment (HipsterBrown) [inline lib/tessel/deployment/lists/javascript.js]: Do we still need this in the blacklist?

Comment (rwaldron) [reply lib/tessel/deployment/lists/javascript.js]: Yes, it's being used for "supports-color" right now

Comment (rwaldron) [reply lib/tessel/deployment/lists/javascript.js]: Oh, hrm... I think maybe it doesn't need to?

**Pull request merged**